### ISSUE TITLE
Insufficient disk quota for Nvidia Modulus apptainer image

### ISSUE THREAD

**Issue by echoi** [itiger-cluster/itiger-cluster.github.io#3]
### Have you checked that your issue isn't already filed?

- [X] I read through [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my issue.
- [X] Yes, I have checked that this issue isn't already filed.

### Bug description

insufficient disk quota for an apptainer image necessary for my work

### How to reproduce the bug

I'm trying to build a sandbox directory on itiger from an apptainer image, nvcr.io/nvidia/modulus/modulus:24.07 that is available on https://catalog.ngc.nvidia.com/orgs/nvidia/teams/modulus/containers/modulus/tags:
```SHELL
apptainer build --sandbox ./nvidia_modulus_24.07/ docker://nvcr.io/nvidia/modulus/modulus:24.07
```
However, my disk quota (20 GB?) was exceeded during the process (see the error message below.)
Can I get more disk space? If not, please suggest a solution. 

BTW, I've tried to create the sandbox directory in `/project` but got the 'disk quota exceeded' error again.
And I don't think I have access to `/scratch`.

### Error messages and logs

```SHELL
FATAL:   While performing build: conveyor failed to get: while fetching image: initializing source oci:/home/echoi2/.apptainer/cache/blob:87bd834a22b77677b878e29168c9ff87bebb3c543c71f572f5ae39fa7a33addd: copying system image from manifest list: writing blob: sync /home/echoi2/.apptainer/cache/blob/oci-put-blob3970410350: disk quota exceeded
```

### What operating system are you using?

Linux

### Where are you seeing the problem on?

Deployed site

**Comment by mayirasharif** [comment]
Try to remove your cache with `rm -rf .cache/pip/` to free up some space. I've talked with one of the admins and your cache folder is over 8 GB--check this with `du -hs .cache/pip/`

What may be happening is that the modules you are installing with `pip` or `pip3` for your container(s) are unintentionally being installed in your home directory, thus passing the quota. 


After this, in your definition files, add all the modules you'd usually install with `pip` or `pip3` in a section `%post`:
![important_info](https://github.com/user-attachments/assets/be08755a-9d4d-4cac-9398-232754c7c3ac)

Let me know if you need more help. Hope this helps.

**Comment by echoi** [comment]
Thanks for the suggestions.
I started building the sandbox after cleaning up the cache folder with `rm -rf .cache/pip/`, the apptainer cache with `apptainer cache clean`, and `.local` folder.
My home directory was still not enough for holding both a sandbox directory and cache files necessary for making it. However, this time, I found that I can save a .sif file in `/project/echoi2`. So, I think I can go from here.
Thanks again for your help.
